blog-posts/1-announcing-swarmion/announcing-swarmion.md

```diff
@@ -0,0 +1,145 @@
+---
+published: true
+title: 'Introducing Swarmion 🐝, a framework for shipping type-safe Serverless microservices at scale'
```

Comment:
Retour de @Sc0ra:

at scale en trop dans le titre ->  Introducing Swarmion :bee: : a Framework for Shipping Type-safe Serverless Microservices

Reply:
`a Framework for` ca sonne bizarre un peu.

```suggestion
title: 'Introducing Swarmion 🐝 : a Type-safe Serverless Microservices Framework'
```